index.bs

```diff
@@ -119,8 +162,18 @@ Sensor use cases</a></cite> documents.
 Security and Privacy Considerations {#security-and-privacy}
 ===================================
 
-There are no specific security and privacy considerations
-beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
+Sensor readings provided by inertial sensors, such as accelerometer, could be used by adversaries
+to exploit various security threats, for example, [=keylogging=], [=location tracking=],
+[=fingerprinting=] and [=user identifying=].
+
+Research papers published by security community, for instance, [[KEYSTROKEDEFENSE]], indicate that
+by throttling the frequency, risks of successful attacks is not fully eliminated, while may greatly
```

Comment:
`by throttling the frequency, risks of successful attacks are not fully eliminated, while throttling may greatly`